frappe/desk/search.py

```diff
@@ -109,7 +109,17 @@ def search_widget(
 	elif not query and doctype in standard_queries:
 		# from standard queries
 		search_widget(
-			doctype, txt, standard_queries[doctype][0], searchfield, start, page_length, filters
+			doctype,
+			txt,
+			standard_queries[doctype][0],
+			searchfield,
+			start,
+			page_length,
+			filters,
+			filter_fields,
+			as_dict,
+			reference_doctype,
+			ignore_user_permissions,
```

Comment:
make these kwargs. It's fairly easy to mess up order and swap two boolean fields.

Reply:
@ankush updated